### PR TITLE
Allow action to run on current release branch.

### PR DESCRIPTION
**Pull request by thst-nordic**
Add `base_branch` but keep main default

tested here: https://github.com/nrfconnect/sdk-zephyr-testing/pull/17

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -6,6 +6,11 @@ inputs:
       type: string
       default: nrfconnect/sdk-nrf
       description: 'target repository for manifest PR'
+    base-branch:
+      required: false
+      type: string
+      default: main
+      description: 'base/target branch for manifest PR'
```
Comment (karhama) [inline action.yml:13]: I believe this should be used after line 78 (in checkout phase).

File: action.yml [hunk 2]
```diff
@@ -6,6 +6,11 @@ inputs:
       type: string
       default: nrfconnect/sdk-nrf
       description: 'target repository for manifest PR'
+    base-branch:
```
Comment (karhama) [inline action.yml:9]: If release branch names match between repos --> alternative to use github.base_ref

Comment (thst-nordic) [reply action.yml:9]: they don't match.

File: action.yml (hunk 1 above)
Comment (thst-nordic) [reply action.yml:13]: added:  https://github.com/actions/checkout/blob/main/README.md?plain=1#L30

File: action.yml (hunk 2 above)
Comment (karhama) [reply action.yml:9]: if they do not match --> needs input.

**Review by karhama — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**